_pkgdown.yaml

```diff
@@ -11,6 +11,17 @@ navbar:
     - icon: fa-github fa-lg
       href: https://github.com/roche/crmPack
 
+articles:
```

Comment:
A little googling makes me think the error is caused by the omission of a new(?) vignette from the articles list, also in `_pkgdown.yaml`.    I've added
```
    - parallel_computing_with_extensions
```
locally and am waiting for the Check action to complete before pushing again, in case there are other errors there that need fixing.